tests/e2e/api/staking.test.ts

```diff
@@ -0,0 +1,43 @@
+// Copyright © Aptos Foundation
+// SPDX-License-Identifier: Apache-2.0
+
+import { AptosConfig, Aptos } from "../../../src";
+import { Network } from "../../../src/utils/apiEndpoints";
+
+// Note: These test against main net and could fail.
```

Comment:
Let's not have unit tests that could fail, they're used for CI

What I suggest is you can probably get a query for all pools, and then verify that you are getting values that match the expected data.

The data in the response is pretty irrelevant since it comes directly from the indexer